components/combobox/themes/express.css

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
```

Comment:
Happy new year 🎉

Reply:
Updated in [908252e](https://github.com/adobe/spectrum-css/pull/1556/commits/908252e02321c7a8fc002e709fcad8fdc6350de6)